service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -150,7 +148,7 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
             .put("document_filename", document.originalDocumentName)
             .build());
 
-        data.remove("amountToPay");
+        removeTemporaryFields(caseDetails, "amountToPay", "displayAmountToPay");
```

Comment:
Possibly could be extracted to the util class as it's noot specific to that controller.

Reply:
Was more thinking it could become part of some sort of base controller class. Although I don't think Lombok's constructor annotations allow for things to be passed to a super class 🤔, will have a look.

Reply:
Doesn't seem like it https://stackoverflow.com/a/29771875 (old answer but doesn't seem to have changed), would have been useful to have a base class which contained stuff like the object mapper ect, ah well, will move to a controller helper class

Reply:
not sure if we have to remove amoutToPay, it can be useful audit information, but anyway sits in payment service